extend/common-interface/folders.md

```diff
@@ -193,11 +193,53 @@ Notice that some of the values might be empty for different workspace backends (
 They will be always present, though.
 
 When exchanging data via workspace, there are couple of differences to loading data into files:
-- Loading to workspaces supports only [storage tables](/storage/tables/), [storage files](/storage/file-uploads/) 
+- Loading to workspaces supports only [storage tables](/storage/tables/), [storage files](/storage/file-uploads/)
 are always saved to the directory structure.
 - The `days` attribute is not supported for filtering table, use `changed_since` instead.
 - [Automatic Incremental Processing](https://help.keboola.com/storage/tables/#automatic-incremental-processing) (also known as Adaptive Input Mapping) is not supported.
 - When used for output mapping, the `columns` of the output table **must be** specified, this can be done either in the [output manifest](/extend/common-interface/manifest-files/#dataouttables-manifests) or in the [output mapping](/extend/common-interface/config-file/#output-mapping--headless-csv).
 
-**Note**: Currently only some combinations of input/output staging storage settings are supported: 
+**Note**: Currently only some combinations of input/output staging storage settings are supported:
 `local<->local`, `local<->s3`, `workspace-snowflake<->workspace-snowflake`, `workspace-redshift<->workspace-redshift`.
+
+## Exchanging Data via File System Workspace
+*Note: this is a preview feature and may change considerably in future.*
+*Note: currently only Azure Blob Storage workspaces (abs-workspace) are supported for this type and those only work with synapse storage backend
+
+The component may also exchange data with a provisioned file workspace (Azure Blob Storage) [using Workspaces](https://keboola.docs.apiary.io/#reference/workspaces).
+This mode of operation can be enabled by setting the **Staging storage input** or **Staging storage output** option
+to **Workspace ABS**. A filesystem workspace is an isolated file storage to which data are loaded before the component job is run (when staging storage input is set)
+and unloaded from when the job finishes (when staging storage output is set).
+The workspace is created just before the job starts and is deleted when the job is terminated.
+
+If this option is enabled, the data and the manifests will be loaded to the azure storage blob container under the data folder similarly to how it does when using the default local filesystem.
+For example, if a file 'test.txt' with ID '12345' is in the input mapping then the file will appear in the storage blob container with URL https://[storage_account_name].blob.core.windows.net/[container-name]/data/in/files/12345_test.txt/12345
+
+In order to write a file for 'source' for the output mapping, write the file to `[containerName]/data/out/files/my-file-name` and it will be exported if it is listed in the output mapping.
```

Comment:
I don't understand what source means here :) 

Perhaps a sample of the OM configuration JSON would help.